cvat-ui/src/utils/opencv-wrapper/histogram-equalization.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { ImageProcessing } from './opencv-interfaces';
+
+export interface HistogramEqualization extends ImageProcessing{
+    processImage: (src:ImageData, frameNumber: number)=>ImageData;
+    restoreImage: ()=>ImageData|undefined;
+}
+
+interface HashedImage{
+    frameNumber: number,
+    frameData: ImageData,
+    timestamp: number,
+}
+
+export default class HistogramEqualizationImplementation implements HistogramEqualization {
+    private readonly bufferSize: number = 20;
+    private cv:any;
+    private histHash: HashedImage[];
+    private currentUnequalized: ImageData | undefined;
```

Comment:
So, finally why do we need to store unequalized image? Is this code outdated?

Reply:
Yes, it is. It should be removed